village/src/components/SmurfForm.js

```diff
@@ -10,9 +12,20 @@ class SmurfForm extends Component {
     };
   }
 
+  addInfo = (URL, entry) => {
+      axios
+        .post(URL, entry)
+        .then(response => console.log(response))
```

Comment:
you can console.log inside the first then and not have to use  2 here if you break it onto  2 lines like so, 
then( response => { 
  console.log(response); 
  this.props.update("http://localhost:3333/smurfs")
})